packages/@webex/plugin-cc/test/unit/spec/services/config/index.ts

```diff
@@ -0,0 +1,200 @@
+import {WebexSDK} from '../../../../../src/types';
+import AgentConfigService from '../../../../../src/services/config';
+import HttpRequest from '../../../../../src/services/core/HttpRequest';
+import {WCC_API_GATEWAY} from '../../../../../src/services/constants';
+import MockWebex from '@webex/test-helper-mock-webex';
+
+describe('AgentConfigService', () => {
+  let agentConfigService: AgentConfigService;
+  let webex: WebexSDK;
+  let mockHttpRequest: HttpRequest;
+  const mockAgentId = 'agent123';
+  const mockOrgId = 'org123';
+  const mockWccAPIURL = WCC_API_GATEWAY;
+
+  beforeEach(() => {
+    webex = new MockWebex({
+      logger: {
+        log: jest.fn(),
+      },
+    });
+
+    webex.internal.device.orgId = mockOrgId;
+
+    mockHttpRequest = HttpRequest.getInstance({webex});
+    mockHttpRequest.request = jest.fn();
+
+    agentConfigService = new AgentConfigService(mockAgentId, webex, mockHttpRequest);
+  });
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  describe('getUserUsingCI', () => {
```

Comment:
There was 4xx,5xx error response handling in the code for which Parv has added the tests. They seem to be missing. Please check and add them back

Reply:
addressed